app/views/admin/loans/content/_form.html.slim

```diff
@@ -10,18 +10,29 @@
 
   = f.error_notification
 
-  - if @loan.division && division_policy(@loan.division).show?
-    = f.input :division
-      .view-element
-        - if @loan.division
+  = f.input :division
+    .view-element
+       - if @loan.division && division_policy(@loan.division).show?
           = link_to @loan.division.name, admin_division_path(@loan.division_id)
-      = f.input_field :division_id, collection: @division_choices, include_blank: false
+       - else
+        = @loan.division.try(:name)
```

Comment:
Indent is not quite right above. I think if/else are one space too far in. Please take a look.